commit 26c1d30b3eb96496024f6650a1b57aa4df88153e
Author: popcornmix <popcornmix@gmail.com>
Date:   Wed Oct 17 16:32:52 2018 +0100

    bcm2708-rpi: Disable txp interrupt unless using vc4-kms-v3d overlay
    
    Signed-off-by: popcornmix <popcornmix@gmail.com>

diff --git a/arch/arm/boot/dts/bcm2708-rpi.dtsi b/arch/arm/boot/dts/bcm2708-rpi.dtsi
index 00160e0b8ab7..131be649e146 100644
--- a/arch/arm/boot/dts/bcm2708-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2708-rpi.dtsi
@@ -89,6 +89,10 @@
 		sound: sound {
 			status = "disabled";
 		};
+
+		txp: txp@7e004000 {
+			status = "disabled";
+		};
 	};
 
 	__overrides__ {
diff --git a/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts b/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts
index 489679637306..f6610c1d70a9 100644
--- a/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts
@@ -141,6 +141,13 @@
 		};
 	};
 
+	fragment@17 {
+		target = <&txp>;
+		__overlay__  {
+			status = "okay";
+		};
+	};
+
 	__overrides__ {
 		cma-256 = <0>,"+0-1-2-3-4";
 		cma-192 = <0>,"-0+1-2-3-4";
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1143_26c1d30b3eb96496024f6650a1b57aa4df88153e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1143_26c1d30b3eb96496024f6650a1b57aa4df88153e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
